.readthedocs.yaml

```diff
@@ -8,4 +8,4 @@ python:
 build:
   os: "ubuntu-22.04"
   tools:
-    python: "3.7"
+    python: "3.12"
```

Comment:
@crenshaw-dev  you could reuse the yaml custom manager to update this too

By removing the "version" from both matchstring it should be able to update any field y any yaml

Reply:
Sure... I'm still trying to learn how datasources and package names work. Looking at it now...